src/metabase/driver.clj

```diff
@@ -107,6 +107,22 @@
 
           Is this property required? Defaults to `false`.")
 
+  (execute-query ^java.util.Map [this, ^Map query]
+    "Execute a query against the database and return the results.
+
+     The query passed in will contain:
```

Comment:
as with the crufty table stuff I'm working on this second paragraph should probably be indented slightly less otherwise it will be formated as a code block in the dox. 

Take a look at it in the instant cheatsheet